web/pages/resources.tsx

```diff
@@ -0,0 +1,489 @@
+      },
+    ],
+  },
+  {
+    teamName: "Volunteer Teams",
+    primaryContact: " Susan DuBois (susan@angelsrescue.org)",
+    contacts: [
+      {
+        email: "volunteers",
+        purpose:
+          "to offer to volunteer on a team or to transport, general volunteering questions",
+      },
+      {
+        email: "recognition",
+        purpose:
+          "to ask that a card be sent to another foster or volunteer going through a difficult time or for going above and beyond",
+      },
+      {
+        email: "fbgroups",
+        purpose:
+          "to be added to one of our internal pages, must be current foster or volunteer",
+      },
+    ],
+  },
+  {
+    teamName: "Social Media & Marketing Teams",
+    primaryContact: "Hilary Van Brunt (hilaryv@angelsrescue.org)",
+    contacts: [
+      {
+        email: "socialmedia",
+        purpose:
+          "to send pictures and video of your foster for posting on our social medial platforms",
+      },
+      {
+        email: "photography",
+        purpose:
+          "to have professional pictures taken of your foster by our volunteer photographers",
+      },
+      {
+        email: "marketing",
+        purpose:
+          "for marketing opportunities and needs, including flyers, brochures, etc.",
+      },
+    ],
+  },
+  {
+    teamName: "Development & Events Teams",
+    primaryContact: "Jackie Spett (jackie@angelsrescue.org)",
+    contacts: [
+      {
+        email: "events",
+        purpose:
+          "for questions about Angels events or to direct people asking about having an event",
+      },
+      {
+        email: "matchinggifts",
+        purpose: "if your company does matching gifts",
+      },
+      {
+        email: "donate",
+        purpose:
+          "donations questions, problems, or to request a thank you card",
+      },
+    ],
+  },
+];
+
+interface TeamProps {
+  team: Team;
+  children: JSX.Element | JSX.Element[];
+}
+
+interface TeamContactProps {
+  contact: string;
+  purpose: string;
+}
+
+function TeamContact({ contact, purpose }: TeamContactProps) {
+  return (
+    <Flex
+      bgColor="tag-primary-bg"
+      flexDir="column"
+      paddingX={4}
+      paddingY={3}
+      borderRadius={10}
+    >
+      <Text fontWeight="600">{`${contact}@angelsrescue.org`}</Text>
+      <Text>{purpose}</Text>
+    </Flex>
+  );
+}
+
+function Team({ team, children }: TeamProps) {
+  return (
+    <Flex
+      w="100%"
+      display="inline-block"
+      direction="column"
+      bgColor="#FFFFFF"
+      border="solid"
+      borderRadius={12}
+      borderWidth="2px"
+      borderColor="#BBBBBB"
+      paddingX={6}
+      paddingY={4}
+      marginTop={[4, 6]}
+    >
+      <Text fontSize="lg" fontWeight="600">
+        {team.teamName}
+      </Text>
+      {team.primaryContact && (
+        <Text fontSize="md" fontWeight="600" marginBottom={3}>
+          {team.primaryContact}
+        </Text>
+      )}
+      <Stack direction="column" gap={3}>
```

Comment:
Apply `marginTop` here instead of `marginBottom` on the primary contact text for spacing consistency between the team name and the children